content/client-lib-development-guide/features.textile

```diff
@@ -2117,7 +2117,7 @@ class EventEmitter<Event, Data>:
   off() // RTE5
   off((Data...) ->) // RTE5
   off(Event, (Data...) ->) // RTE5
-  emit(Event, Data...)  // RTE6
+  emit(Event, Data...)  // internal, RTE6
```

Comment:
Maybe to make _internal_ specifier a part of the IDL? Just thoughts, can be ignored.

Reply:
I think it's a good idea but not one I really want to address here - this PR is quite big already and I'm just following the existing habit of marking internal things with a comment. Personally I'm not sure whether we should even have internal methods in the IDL, but that's for another day 😄